redirect.go

```diff
@@ -191,11 +205,34 @@ func (r *Redirect) To(location string) error {
 // If you want to send queries or params to route, you should use config parameter.
 func (r *Redirect) Route(name string, config ...RedirectConfig) error {
 	// Check config
-	cfg := RedirectConfig{}
+	cfg := RedirectConfig{CookieConfig: CookieConfigDefault}
```

Comment:
since the part with the configurations has become larger, we can move this to a private function for creating the configuration, so that it becomes more usable again